### ISSUE TITLE
Please create wiki or add to ReadME: Info about IDE Key and Trigger Values

### ISSUE THREAD

**Issue by TonyGravagno** [BrianGilbert/xdebug-helper-for-firefox#42]
The configuration says:

> Select your IDE to use the default sessionkey or choose other to use a custom key.

Then we have:

> If your xdebug is configured with a unique trace trigger key, set that value here
> If your xdebug is configured with a unique profile trigger key, set that value here.

I can't find any information for this extension about what configuration is required in the IDE that corresponds to these values. There's nothing here about what values to use or why.

### Trigger values

The Trace Trigger Value or Profile Trigger Value are sent from the browser to the server when tracing or profiling are enabled. In php.ini, tell xdebug to start when the request includes a trigger, and also what trigger should be used. Examples:

Set the Trace Trigger Value for the extension to "Debugging" and the Profile Trigger Value to "Profiling".  
Then set php.ini values as follows:

```
xdebug.start_with_request=trigger
xdebug.trigger_value=Debugging,Profiling
```

The actual values can be anything simple (avoid spaces and alphanumerics), as long as the client and server settings agree on the values.

Xdebug help for [start_with_request=trigger](https://xdebug.org/docs/all_settings#start_with_request#trigger).  
Xdebug help for [trigger_value](https://xdebug.org/docs/trace#trigger_value).

Prior to Xdebug 3.1, trigger_value only supports one value.

### IDE Key

[TL;DR] Let's preface with a [quote](https://stackoverflow.com/questions/71841055/xdebug-works-with-phpstorm-but-not-on-vscode#comment126952438_71841055) from Derick Rethans (author of Xdebug) : "the IDE key is irrelevant".

Here is an [example of logging that shows the idekey](https://xdebug.org/docs/trace#log). The documentation for that logging, specifically [Connection Initialization](https://xdebug.org/docs/dbgp#connection-initialization) which includes the XML init tag, describes this as the DBGP_IDEKEY environment variable, and says "This value may be empty."

**IDE Key with VSCode example:**

The IDE Key is pre-defined in this extension for some popular IDEs, but not for other popular IDEs. In particular, Visual Studio Code is not yet in the list. For this and other undefined IDEs, use the "other" option in extension config.

The commonly accepted value for VSCode is `VSCODE`, all caps.  See [this discussion](https://github.com/mac-cain13/xdebug-helper-for-chrome/issues/89) in the Chrome extension on which this Firefox extension is based. Other values can be used, though a developer should have a good reason and research the topic.

To get VSCode to agree with the IDE Key in this extension, the VSCode environment can be configured with specific idekey values [source](https://github.com/xdebug/vscode-php-debug/issues/403#issue-610419524). See configuration documentation for other IDEs that support Xdebug for equivalent configuration details.

launch.json : 

```
"configurations": [
   {
      "name": "Debug Configname",
...
      "env": {
	      "XDEBUG_CONFIG": "idekey=VSCODE",
	      "DBGP_IDEKEY": "VSCODE"
      },
   }
```

----

The text here isn't being suggested as verbatim content for the Readme or wiki, but as material that can be included to help people to understand and use this extension.

I haven't even fully verified all of the detail. Let's check it out, verify it, modify it here, and then publish the final verified information somewhere.

Thanks.

**Comment by mortona42** [comment]
With xdebug set to start with trigger, the debugger needs to get a XDEBUG_TRIGGER value.
I can set this in the url, but it would be nice to use this extension to set it as a cookie.

**Comment by TonyGravagno** [comment]
As I understand it, the trigger value is currently sent in the header. If you want it in a cookie, I believe that would be an enhancement request, and should be in a different issue. I created this ticket to request documentation for how the extension currently works. If/when that's in place and new features are added related to this area, we'll need to ensure the doc is updated with the current functionality.

In summary : I suggest you create a new ticket. If this extension is already supposed to put the value in a cookie and it doesn't that's a bug, otherwise this is an enhancement request. When there is info about that functionality, yeah, let's add it to this ticket.

**Comment by BrianGilbert** [comment]
the settings popup has info about this, 1.0.10 (pending review by mozilla) has a link to settings from the address bar popup.
